hadoop-hdfs-project/hadoop-hdfs-client/src/main/java/org/apache/hadoop/hdfs/web/WebHdfsFileSystem.java

```diff
@@ -478,7 +478,7 @@ private Path makeAbsolute(Path f) {
     return f.isAbsolute()? f: new Path(workingDir, f);
   }
 
-  static Map<?, ?> jsonParse(final HttpURLConnection c,
+  public static Map<?, ?> jsonParse(final HttpURLConnection c,
       final boolean useErrorStream) throws IOException {
```

Comment:
Is it made public only for test?
If so please add VisibleForTesting Annotation

Reply:
OK. Thanks for your review.